action.yaml

```diff
@@ -32,7 +35,7 @@ runs:
 
     - name: Create Release
       env:
-        GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
```

Comment:
`env` here is from the `env` block in the workflow yaml. not quite sure it can be accessed from here

Reply:
This is exactly the issue - thank you